e2e/tests/api_wa_tests/1v1_unspec_ga_wa_specific_access_test.js

```diff
@@ -3,10 +3,10 @@ const mpScenario = 'ONE_V_ONE';
 
 let civilCaseReference, gaCaseReference;
 
-Feature(' GA - WA Specific Access @api-wa');
+Feature(' GA - WA Specific Access @api-ga-wa');
```

Comment:
@deepthidoppalapudihmcts it is supposed to be @api-wa . please revert these changes